packages/fluentui/react-northstar/bundle-size-auditor.config.js

```diff
@@ -0,0 +1,32 @@
+// @ts-check
+const path = require('path');
+const fs = require('fs');
+
+module.exports = /** @type {import('@fluentui/scripts-bundle-size-auditor').Config} */ ({
+  createFixtures: config => {
+    const packageName = '@fluentui/react-northstar';
```

Comment:
react and react-northstar have different logic for fixture creation so I created the api like this - in future it can be simplified if necessary ( for example to only provide directory for fixtures + some kind of pattern )